db/migrate/20230316034707_disable_inherits_tax_category_from_per_order_enterprise_fees.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class DisableInheritsTaxCategoryFromPerOrderEnterpriseFees < ActiveRecord::Migration[6.1]
+  class EnterpriseFee < ApplicationRecord
+    has_many :calculator, class_name: "Spree::Calculator", foreign_key: :calculable_id
```

Comment:
From what I can understand in the last comment, you've updated this line. 
I think that using `as: :calculable` as [in the model](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/models/concerns/calculated_adjustments.rb#L9) would have achieved the same result (rails would guess "calculable_id" from "calculable").
But it sounds like this works too 👍 .

I think sometimes it's actually simpler and safer to just build the SQL directly for a migration, rather than try to re-create the model.

Reply:
>I think that using as: :calculable as [in the model](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/models/concerns/calculated_adjustments.rb#L9) would have achieved the same result (rails would guess "calculable_id" from "calculable").

@dacook that's correct, but the problem is with calculable_type. The SQL query will have the two following parameters 
1. calculable_id: will have the correct id of the enterprise fee
2. calculable_type: will be set to `DisableInheritsTaxCategoryFromPerOrderEnterpriseFees::EnterpriseFee`, which is wrong ( it must be `EnterpriseFee`)